src/templates/dotnet/executeDotnetTemplateCommand.ts

```diff
@@ -60,25 +61,26 @@ async function getFramework(context: IActionContext, workingDirectory: string |
         }
 
         // Prioritize "LTS", then "Current", then "Preview"
```

Comment:
Looks good to me other than the test failures. The Windows timeouts are probably unrelated, but I think Mac was working just fine before this change. Unfortunately nothing jumps out at me as to _why_ they're failing

Reply:
Sounds good. I'll figure out what's up.

Reply:
The Windows failure is probably transient; one of the builds (for `bmw/netVersions2`) succeeded, one (for the PR) failed. The Mac one is definitely suspicious; I'm looking into it now.

Reply:
Still need to work out why, but despite the "Use .NET SDK..." tasks, the .NET version info visible in the test is as follows:
```
2.1.507
2.1.302 [/Users/runner/.dotnet/sdk]
2.1.403 [/Users/runner/.dotnet/sdk]
2.1.507 [/Users/runner/.dotnet/sdk]
```

Reply:
In contrast, on Linux, the versions and paths above match. So it looks like something is up with `PATH`...on Mac...imagine that.